packages/hardhat-plugin/README.md

```diff
@@ -0,0 +1,87 @@
+# Unlock Hardhat plugin
+
+_Unlock hardhat plugin provides a set of tasks and scripts to interact directly with the [Unlock Protocol](https://unlock-protocol.org)_
+
+
+## Installation
+
+```bash
+yarn install @unlock-protocol/hardhat-plugin
```

Comment:
I assume this means we should publish it! Do you have access to npm? Let's talk about it during our 1/1

Reply:
yes its published already ;)